code/Model/SiteTree.php

```diff
@@ -3157,7 +3173,7 @@ public function provideI18nEntities()
      */
     public function getParentType()
     {
-        return $this->ParentID == 0 ? 'root' : 'subpage';
+        return $this->ParentID === 0 ? 'root' : 'subpage';
```

Comment:
```suggestion
        return !$this->ParentID ? 'root' : 'subpage';
```
`===` will break if $this->ParentID is `null` for whatever reason